app/bundles/LeadBundle/Entity/LeadListRepository.php

```diff
@@ -1223,10 +1271,22 @@ public function getListFilterExpr($filters, &$parameters, QueryBuilder $q, $not
                             $table  = 'email_stats';
                             $column = 'email_id';
                             break;
+                        case 'device_type':
+                            $table  = 'lead_devices';
+                            $column = 'device';
+                            break;
+                        case 'device_brand':
+                            $table  = 'lead_devices';
+                            $column = 'device_brand';
+                            break;
+                        case 'device_os':
+                            $table  = 'lead_devices';
+                            $column = 'device_os_name';
+                            break;
                     }
 
                     $subExpr->add(
-                        $subQb->expr()->in(sprintf('%s.%s', $alias, $column), $details['filter'])
+                        $subQb->expr()->in(sprintf('%s.%s', $alias, $column),  "'".implode("','", $details['filter'])."'")
```

Comment:
The `in` method works correctly only with second param type of `array`, not `string`. (if the array contains more than 1 items)